Source/WebKit/WebProcess/Plugins/PluginView.cpp

```diff
@@ -428,8 +428,15 @@ void PluginView::paint(GraphicsContext& context, const IntRect& /*dirtyRect*/, W
     if (paintRect.isEmpty())
         return;
 
-    if (m_transientPaintingSnapshot)
-        m_transientPaintingSnapshot->paint(context, m_plugin->deviceScaleFactor(), frameRect().location(), m_transientPaintingSnapshot->bounds());
+    if (m_transientPaintingSnapshot) {
+        if (!context.platformContext()) {
```

Comment:
Do we hit this for the "update control tints" and "detecting largest contentful paint" fake paints?

Reply:
No since those pass a `NullGraphicsContext` which early returns in this function when checking `context.paintingDisabled()`.